src/rdb/rdb.c

```diff
@@ -346,6 +356,7 @@ rdb_stop(struct rdb *db)
 	vos_pool_close(db->d_pool);
 	rdb_kvs_cache_destroy(db->d_kvss);
 	ABT_cond_free(&db->d_ref_cv);
+	ABT_mutex_free(&db->d_raft_mutex);
 	ABT_mutex_free(&db->d_mutex);
 	D_FREE(db);
 }
```

Comment:
In rdb.c, there's also rdb_is_leader and rdb_get_leader who also need to take d_raft_mutex.